app/controllers/books_controller.rb

```diff
@@ -35,6 +37,10 @@ def create
     end
   end
 
+  def reviewed?(book)
+    Review.where(user_id: current_user.id, book_id: book.id).any?
+  end
```

Comment:
This should really be a method on the Book class. It would be ideal to be able to call `@book.reviewed?` and get a true or false value. Move this method to that class and write a unit test for it!